docs/applications/containers/introduction-to-microservices-with-kubernetes-and-gitlab/index.md

```diff
@@ -0,0 +1,210 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'Introduction to Microservices With Kubernetes and Gitlab'
+keywords: ['kubernetes','k8s', 'Gitlab', 'Git', 'definitions']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2019-08-21
+modified_by:
+  name: Linode
+title: "Introduction to Microservices With Kubernetes and Gitlab"
+contributor:
+  name: Linode
+external_resources:
+- '[Gitlab](https://about.gitlab.com/https://about.gitlab.com/)'
+- '[Kubernetes](https://kubernetes.io/)'
+- '[Helm](https://helm.sh/)'
+- '[Troubleshooting Gitlab](https://docs.gitlab.com/charts/troubleshooting/index.html)'
+
+---
+This guide is the first part of a three-part series on using microservices with Kubernetes and Gitlab. Parts two and three will give a high level overview of what microservice applications on Kubernetes are doing under the hood.
+
+In this guide you will:
```

Comment:
I broke this out into a "In this guide you will" section -  but the intro now could use a little work.